uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/FieldsPopupElementView.qml

```diff
@@ -73,9 +73,9 @@ ListView {
                 // Checks to see if the formatted value is a hyperlink.
                 // If it is, it will modify the text to be a clickable link
                 // displayed as `View`.
-                if (text.startsWith("http")) {
+                if (text.toLowerCase().startsWith("http")) {
                     let link = text;
-                    text = `<a href="${link}" target="_blank">View</a>`;
+                    text = `<a href="${link}">View</a>`;
```

Comment:
I reviewed html documentation and `target="_blank"` is for specifying if it should open in a new window but that doesn't apply since this isn't a browser. Simplified this as a result.